models/core/eligibility.yml

```diff
@@ -0,0 +1,8 @@
+version: 2

```

Comment:
Similar comment from sources.yml,  I think it would be nice to have the descriptions listed out here for the documentation.